packages/core-data/src/entities.js

```diff
@@ -175,6 +175,14 @@ export const rootEntitiesConfig = [
 		baseURLParams: { context: 'edit' },
 		key: 'plugin',
 	},
+	{
+		label: __( 'Navigation' ), // Block based Navigation Menus
```

Comment:
Is it a post type?

Reply:
It's a custom post type yes. That's why it's 

```
wp.data.select('core').getEntityRecords('postType','wp_navigation')
```

Reply:
It's a custom post type yes. That's why it's 

```
wp.data.select('core').getEntityRecords('postType','wp_navigation')
```

Reply:
In this case this needs a follow up to replace all the `getEntityRecords('postType','wp_navigation')` calls for the `getEntityRecords('root','navigationMenus')`. 

But thinking more about this – why do we introduce two different ways to refer to the same thing? Couldn't we just hardcode a shortcut that would work with the `postType, wp_navigation` pair?